hardhat-reference.config.ts

```diff
@@ -28,15 +28,47 @@ const config: HardhatUserConfig = {
   solidity: {
     compilers: [
       {
-        version: "0.8.7",
+        version: "0.8.14",
```

Comment:
This version was selected for the reference contracts deliberately as it's a "stable" compiler version (in contrast to more recent versions which lean into the yul IR pipeline)

Reply:
Makes sense.